src/Elcodi/Admin/CoreBundle/Controller/Abstracts/AbstractAdminController.php

```diff
@@ -163,23 +166,24 @@ protected function getResponse(
      *
      * This method takes into account the type of the request ( GET or POST )
      *
-     * @param Request $request     Request
-     * @param string  $redirectUrl Redirect url
+     * @param Request $request      Request
+     * @param string  $redirectPath Redirect path
      *
      * @return mixed Response
      */
-    protected function getOkResponse(Request $request, $redirectUrl = null)
-    {
-        if (null === $redirectUrl) {
-            $redirectRoute = $request->headers->get('referer');
-        } elseif ($redirectUrl instanceof Closure) {
```

Comment:
I have not found anywhere where this is used. Anyway, it makes no sense for me to check if a parameter called "redirectPath" as we have now is a closure.